diff --git a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
--- a/cmake/CMakeLists.txt
+++ b/cmake/CMakeLists.txt
@@ -335,7 +335,8 @@ set(STANDARD_PACKAGES
   PYTHON
   QEQ
   QMMM
   QTB
+  RAPTOR
   REACTION
   REAXFF
   REPLICA
@@ -458,5 +459,7 @@ pkg_depends(ML-RUNNER KSPACE)
 pkg_depends(RHEO BPM)
 pkg_depends(APIP ML-PACE)
-
+pkg_depends(RAPTOR MOLECULE)
+pkg_depends(RAPTOR KSPACE)
+
 # detect if we may enable OpenMP support by default
 set(BUILD_OMP_DEFAULT OFF)
